torchbox/torchbox.com
# Add styles for gists

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1528430831)

### Description of Changes Made

Styles for codehilite have been added to the build but these only kick in when a programming language is added in code block streamfield. 

When a link to a github gist is added via a raw HTML streamfileld then different classes are used for styling, not `codehilite`.

I've found a github repo where `gist`'s are styled so have added that to the build. After doing so there are no colour contrast issues being reported by aXe.

### How to Test

- Add a github gist to a page and check the colour contrast (if you have prod data then http://localhost:8000/blog/how-do-you-spell-cabbages/ should work)

### Screenshots

<details>
  <summary>Before</summary>


![Screenshot 2024-07-24 at 10 45 36](https://github.com/user-attachments/assets/1f952da6-c82a-47ea-941f-fecb16e20845)

</details>

<details>
  <summary>After</summary>


![Screenshot 2024-07-24 at 10 45 09](https://github.com/user-attachments/assets/f6485dc6-e3ca-4c3c-adad-dc73f996603a)


</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
